website/docs/reference/resource-properties/arguments.md

```diff
@@ -23,9 +32,37 @@ macros:
 
 </File>
 
+</TabItem>
+
+<TabItem value="functions">
+
+<File name='functions/<filename>.yml'>
+
+```yml
+
+version: 2
+
+functions:
+  - name: <function name>
+    arguments:
+      - name: <arg name>
+        data_type: <string> # warehouse-specific
+        description: <markdown_string>
+
+```
+
+</File>
+
+</TabItem>
+</Tabs>
+
 ## Definition
 
-The `arguments` property is used to define the parameters that a macro can accept. Each argument can have a `name`, `type`, and `description`.  You can add `arguments` to a [macro property](/reference/macro-properties), which helps in documenting the macro and understanding what inputs it requires.
+The `arguments` property is used to define the parameters that a macro or function can accept. Each argument can have a `name`, `type` (for macros) or `data_type` (for functions), and `description`.  
```

Comment:
the `arguments` property is distinct for macros vs. functions - i recommend making these two different pages

Reply:
<img width="271" height="126" alt="Screenshot 2025-10-08 at 6 45 17 AM" src="https://github.com/user-attachments/assets/a9b314fc-e64b-4e40-9ec5-484bcebfde9b" />

and making a new section "for functions" with an arguments page nested